packages/fetch-and-compile/lib/fetch.ts

```diff
@@ -118,13 +129,16 @@ async function tryFetchAndCompileAddress(
       break;
     }
     //set up the config
-    let externalConfig: Config = config.with({
+    let externalConfig: Config = Config.default().with({
```

Comment:
This could be created before being used in this loop.

Reply:
I am confused, no it can't?  It depends on `options`, which has only just been determined.

Reply:
You are not confused, sir! I was wrong.